Assets/ScriptableObjects/Items/AuctionStages/ExtensionsWeighted.asset

```diff
@@ -16,7 +16,7 @@ MonoBehaviour:
   items:
   - {fileID: 11400000, guid: a0f539379f7384445bc1535317457198, type: 2}
   - {fileID: 11400000, guid: bb334224a9209984b98cc009f333ca9d, type: 2}
-  - {fileID: 11400000, guid: bb334224a9209984b98cc009f333ca9d, type: 2}
+  - {fileID: 11400000, guid: 51bb200d04523dd469ed0d8fc736ee64, type: 2}
```

Comment:
These should be automatically populated by the stuff in StaticInfo, did you change this file directly?

Reply:
No, it changes the population automatically, it probably became like this after testing